examples/onnxruntime/optimization/multiple-choice/run_swag.py

```diff
@@ -164,17 +164,35 @@ class OptimizationArguments:
     )
 
 
+@dataclass
+class OnnxExportArguments:
+    """
+    Arguments to decide how the ModelProto will be saved.
+    """
+
+    use_external_data_format: bool = field(

```

Comment:
I don't think `use_external_data_format` should be an argument here as it will not have any impact on the model onnx [export](https://github.com/huggingface/transformers/blob/main/src/transformers/onnx/convert.py#L173). Why not add an `ort_config` attribute to our `ORTModel` with the corresponding information (+  `opset`) and use it in when instantiating our `ORTOptimizer` / `ORTQuantizer` ?

Reply:
That's a good point! Indeed if the original export exceeds 2GB, there would be no need to consider the export after the optimization or the quantization. I will add the attribute in ORTModel as you suggest, thx!

Reply:
My point is that it could be misleading for users as they could think that this will impact the model ONNX export (before optimization) so I think we should specify it. Also I think it could make sense to add an `use_external_data_format` attribute  to our `ORTModel` and save it in our `ORTConfig`  in order to use it in case `use_external_data_format` is not specified when instantiating an `ORTOptimizer` / `ORTQuantizer` as it could be a better default value than `False`, in any case this should be done on a separate PR